src/lib/components/data-converter-port-settings.svelte

```diff
@@ -1,22 +1,28 @@
 <script lang="ts">
+  import Input from '$lib/holocene/input/input.svelte';
   import { translate } from '$lib/i18n/translate';
   export let port = '';
 </script>
 
 <div class="my-4">
-  <h3 class="text-lg" data-testid="data-encoder-port-title">
+  <label
```

Comment:
Can we use `<Input />`'s `label` prop here?

Reply:
This would be my preference 😂 

```svelte
  <Input
    label={translate('data-encoder', 'port-title')}
    placeholder={translate('data-encoder', 'port')}
    id="data-encoder-port-input"
    bind:value={port}
    hintText={translate('data-encoder', 'port-info')}
  />
```

<img width="761" alt="Screen Shot 2023-08-29 at 11 32 41 AM" src="https://github.com/temporalio/ui/assets/11775628/4cccdd04-de73-4171-ab90-9492bb29c0f3">